client-app/shared/wishlists/components/wishlist-line-items.vue

```diff
@@ -77,6 +77,9 @@ defineProps<IProp>();
 const validationErrors = ref<ValidationErrorType[]>([]);
 
 function addToCartDisabled(item: PreparedLineItemType) {
+  if (item.actualPrice?.amount === 0 || item.listPrice?.amount === 0) {
```

Comment:
is the behavior the same for physical and digital products?

Reply:
Yes, right, it was discussed with @Lenajava1 . I've just changed the task description accordingly.

![image](https://github.com/VirtoCommerce/vc-theme-b2b-vue/assets/18286430/f2ae1004-5d40-4a37-b2ad-16d59a447381)